src/components/App/App.jsx

```diff
@@ -1,21 +1,33 @@
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import HeaderBar from '../HeaderBar/HeaderBar';
 import MainContentWrapper from '../MainContentWrapper/MainContentWrapper';
 import { BehandlingerProvider } from '../../context/BehandlingerContext';
 import { InnrapporteringProvider } from '../../context/InnrapporteringContext';
+import { AuthProvider, AuthContext } from '../../context/AuthContext';
 import { withContextProviders } from '../../context/withContextProviders';
 import './App.css';
 import 'reset-css';
+import fetchIntercept from 'fetch-intercept';
 
-const App = withContextProviders(
-    () => (
+const App = withContextProviders(() => {
+    const { setUserLoggedOut } = useContext(AuthContext);
+
+    useEffect(() => {
+        fetchIntercept.register({
+            responseError: error => {
+                setUserLoggedOut();
+                return Promise.reject(error);
+            }
+        });
+    }, []);
```

Comment:
Denne får jeg ikke til å fungere når BE returnerer 401 eller 500 (eller noen andre statuskoder jeg har prøvd). Testet med å sjekke alle responser for 401, og det funker fint:
```javascript
useEffect(() => {
    fetchIntercept.register({
        response: res => {
            if (res.status === 401) {
                setUserLoggedOut();
            }
            return Promise.resolve(res);
        }
    });
}, []);
```